### PR TITLE
LPS-123917 Traffic referral detail backend mock

### PR DESCRIPTION
**Pull request by darquesdev**
```json
[
    {
       "name":"referral",
       "share":"64,5",
       "details":{
          "referringPages":[
             {
                "url":"https://www.liferay.com/resources/ebooks/Becoming+a+Digital+Business-4+Common+Enterprise+Challenges+Conquered",
                "traffic":125461
             },
             {
                "url":"https://www.liferay.com/resources/whitepapers/B2B+E-Commerce+RFP+Kit",
                "traffic":85485
             },
             {
                "url":"https://www.liferay.com/resources/whitepapers/6+Tactics+to+Modernize+Your+Intranet",
                "traffic":84564
             },
             {
                "url":"https://www.liferay.com/resources/case-studies/materion-case-study",
                "traffic":5846
             },
             {
                "url":"https://www.liferay.com/web/l/a1-hrvatska-case-study",
                "traffic":3521
             },
             {
                "url":"https://www.liferay.com/resources/case-studies/excellus-case-study",
                "traffic":2513
             },
             {
                "url":"https://www.liferay.com/resources/case-studies/terres-inovia-case-study",
                "traffic":2200
             },
             {
                "url":"https://www.liferay.com/resources/case-studies/vodafone-business",
                "traffic":1230
             },
             {
                "url":"https://www.liferay.com/web/guest/resources/case-studies/agia",
                "traffic":1100
             },
             {
                "url":"https://www.liferay.com/resources/case-studies/vitality-case-study",
                "traffic":100
             }
          ],
          "referringDomains":[
             {
                "url":"http://youtube.com/",
                "traffic":17985230
             },
             {
                "url":"http://www.google.com/",
                "traffic":12218030
             },
             {
                "url":"http://microsoft.com/",
                "traffic":9062949
             },
             {
                "url":"http://linkedin.com/",
                "traffic":4601453
             },
             {
                "url":"https://www.liferay.com",
                "traffic":253399
             }
          ]
       },
       "title":"Referral",
       "value":24225897,
       "helpMessage":"This is the number of page views generated by people coming to your page from other sites which are not search engine pages or social sites."
    },
    {
       "name":"social",
       "share":"0,0",
       "title":"Social",
       "value":0,
       "helpMessage":"This is the number of page views generated by people coming to your page from social sites."
    },
    {
       "name":"paid",
       "share":"0,0",
       "title":"Paid",
       "value":0,
       "helpMessage":"This is the number of page views generated by people that find your page through Google AdWords."
    },
    {
       "name":"direct",
       "share":"0,0",
       "title":"Direct",
       "value":0,
       "helpMessage":"This is the number of page views generated by people arriving directly to your page."
    },
    {
       "name":"organic",
       "share":"0,0",
       "title":"Organic",
       "value":0,
       "helpMessage":"This is the number of page views generated by people coming from a search engine."
    }
 ]
```

### CODE REVIEW

**Comment by dgarciasarai** [comment]
Thanks @darquesdev! I'm going to deploy it to try with my frontend commits. If everything goes well, I'm going to send the frontend changes to Frontend Infra to review and forward :) Let's go!

**Review by dgarciasarai — APPROVED** [review]
It's working! 🚀 ❤️ 

* PieChart:

Screenshot 2020-11-30 at 17 58 22 

* Referral Detail:

Screenshot 2020-11-30 at 18 03 28

**Comment by dgarciasarai** [comment]
Frontend changes sent to Frontend Infra: https://github.com/liferay-frontend/liferay-portal/pull/573.

**Comment by dgarciasarai** [comment]
This pull request contains the necessary commits to test the Referral traffic channel detail view. cc: @carlosbrichete

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetTrafficSourcesMVCResourceCommand.java [hunk 1]
```diff
@@ -135,6 +139,57 @@ protected void doServeResource(
 		}
 	}
 
+	private ReferralTrafficDetails _getReferralTrafficDetails() {
```
Comment (cgoncas) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetTrafficSourcesMVCResourceCommand.java:142]: If you want to add this mock, that is totally fine, please, add a FF.

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by darquesdev** [comment]
https://github.com/liferay-tango/liferay-portal/pull/634

**Pull request closed without merging**